src/content/other/atomics.mdx

```diff
@@ -1,11 +1,15 @@
 ---
 title: Nuclear Power
 description: Fission, fusion—what’s up with nuclear power
```

Comment:
We need a way to customize the punctation. This should have a "?" at the end, but doesn't because it _sometimes_ renders as "?.".